turned src/env.py to a normal .env file

ShoaibNagy opened the pull request:
I created an ENV file named ".env". I modified .gitignore, requirements.txt, block.py, cache.py, network.py, record.py, dnsserver.py, and proxyserver.py. .gitignore was modified because .env was repeated three times. block.py was edited due to some unnecessary modifications. The same goes for the rest Python files and requirements.txt.

ShoaibNagy committed:
First Task

ShoaibNagy committed:
First task 2nd trial

ShoaibNagy committed:
Delete .env

ShoaibNagy committed:
Created .env file and edited python files
I created a .env file for variables and edited .gitignore since .env was
repeated three times. I edited all the python files so that they have
the same code before editing them for the first time. Only python files
which import env.py were edited by importing load_dotenv from dotenv and
os and typing the code so that the old code and the new one perform the
same function.

ShoaibNagy committed:
README.md was Modified

ShoaibNagy committed:
requirements.txt was modified

ShoaibNagy committed:
requirements.txt and README.md were modified

ShoaibNagy committed:
Merge branch 'dev' of https://github.com/Developer24032/dns-server into dev
I wanted to include the commit about both requirements.txt and README.md mofications in the
pull request.

ShoaibNagy committed:
LEVEL in README.md was modified

ShoaibNagy committed:
Merge branch 'my-task' into dev
I made other modifications in README.md

ShoaibNagy committed:
Other modifications on README.md

ShoaibNagy committed:
Merge branch 'my-task' into dev
The indentation of overview title's description has been reduced

ahmed-elbehairy7 reviewed (changes requested):
السلام عليكم و رحمة الله وبركاته

الشغل اللهم بارك بسطني جدا حقيقي حقيقي جزاك الله كل خير فعلا جميل...
عندي بس تعديل بسيط جدا جدا تظبطه...
الload_dotenv مش محتاج انك تعملها في كل ملف، مجرد ما بتعملها مرة واحدة بتسمع في السيستم كله، ف هنعملها في الentrypoint بتاعت السيرفر يعني في ملف src\__main__.py ده الملف اللي السيرفر بيبدأ بيه..

![image](https://github.com/user-attachments/assets/847c3d0b-7279-4712-9539-f3ae16eab559)

ده شكل ملف ال__main__.py هتحط الload_dotenv كأول حاجة تحصل في فنكشن الmain

بس كده جزاك الله كل خير يشعيب ❤️

ahmed-elbehairy7 commented on test/records/test_record.py:
خلي بالك انت هنا بردو لعبت في حجات بس اشطا يعني انا قولت انبهك بس لكن في المجمل جزاك الله كل خير طبعا على الشغل الرائع

ShoaibNagy committed:
Python Files were modified

I modified src/records/block.py, src/records/cache.py, src/records/network.py, and src/records/record.py by removing 'from dotenv import load_dotenv' and 'load_dotenv' from them and put those codes in src/__main__.py.

ShoaibNagy committed:
test/records/test_record.py is modified

The import that starts with 'ffrom' was chenged to from
